_component_design/selector-strip.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Selector Strip
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  Selector Strips allow the user to make a selection from a group of options. This pattern is a more visually prominent alternative to check boxes (multi-select) or radio buttons (single-select).
+preview-image: preview-images/selector-strip.svg
+status: stable
+resource: true
+last-modified: 2017-10-29
+---
+
+{% include toc.html %}
+
+## Overview
+
+<div class="hxRow">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+### When to use
+
+Use a selector strip on forms requiring a user to select from a small group of action. Selector strips function similarly to [radio buttons]({{site.baseurl}}/components/radio-buttons.html) or [check-boxes]({{site.baseurl}}/components/checkboxes.html) depending on the variation used; single-select or multi-select. Selector strips are best used in situations where the selection alters the data options presented on the page, or when seeking to more visual prominence to group of options.
+
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Best practice
+
+Selector strips function best when:
+
+- The labels inside the strip are short (1-3 words long max) so the number of options can fit horizontally on a small screen.
+- checkboxes or radio buttons do not make sense visually.
+- the options are important to the rest of user’s workflow and deserve a more prominent visual treatment. 	
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the selector strip pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/selector-strip/select-strip-hero.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Default layout
+
+<div class="hxRow">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+The default layout has two main styles: Single-Select and Multi-Select. In both layouts the unselected options have a white background with cyan text & selected options have a cyan background with white text. 
```

Comment:
lowercase for `single-select` and `multi-select`

Reply:
updated.